assets/lokomotive-kubernetes/bootkube/conditional.tf

```diff
@@ -14,6 +14,7 @@ resource "local_file" "calico" {
     network_ip_autodetection_method = var.network_ip_autodetection_method
     pod_cidr                        = var.pod_cidr
     enable_reporting                = var.enable_reporting
+    calico_metadata_cidrs           = var.calico_metadata_cidrs
```

Comment:
Maybe call this `calico_blocked_metadata_cidrs`? Otherwise the variable is not very descriptive.

Reply:
Also, why `calico`? I'd remove `calico` from the var name. If you find it important to explain the implementation details, I'd mention them either in the var's description or in a comment.

Reply:
I added `calico`, as it is passed to the calico chart later on. `blocked_metadata_cidrs` then?

Reply:
How about `blocked_cidrs`? AFAICT there is nothing special to metadata endpoints in this var.

Reply:
I changed it to `blocked_metadata_cidrs` then.